src/main/java/baseball/number/GameNumber.java

```diff
@@ -0,0 +1,43 @@
+package baseball.number;
+
+public class GameNumber {
+    private String value;
+    private GameNumber(String number) {
```

Comment:
필드와 메소드, 메소드와 메소드 사이에 빈 라인을 추가해주세요 🙂
```suggestion
    private String value;
    
    private GameNumber(String number) {
```